### PR TITLE
[backend] config import error fix 

### PR DESCRIPTION
**Pull request by ann-aot**
config import error fix 
image

### CODE REVIEW

**Review by guru-aot — APPROVED** [review]
LGTM

File: sources/packages/backend/apps/test-db-seeding/src/test-db-seeding.module.ts [hunk 1]
```diff
@@ -25,6 +26,7 @@ import {
     InstitutionUserService,
     InstitutionHelperService,
     UserTypeRoleHelperService,
+    ConfigService,
```
Comment (andrewsignori-aot) [inline sources/packages/backend/apps/test-db-seeding/src/test-db-seeding.module.ts:29]: Please import the `ConfigModule` instead.
@guru-aot FYI

**Review by andrewsignori-aot — COMMENTED** [review]
Please see the comment 😉

**Review by andrewsignori-aot — APPROVED** [review]
👍

**Pull request merged**